libraries/AP_DDS/README.md

```diff
@@ -179,14 +178,10 @@ cd ardupilot
 source /opt/ros/humble/setup.bash
 # Find the IDL file
 find /opt/ros/$ROS_DISTRO -type f -wholename \*builtin_interfaces/msg/Time.idl
-# Create the directory in the source tree if it doesn't exist
-mkdir -p libraries/AP_DDS_Client/Idl/builtin_interfaces/msg/
+# Create the directory in the source tree if it doesn't exist similar to the one found in the ros directory
+mkdir -p libraries/AP_DDS/Idl/builtin_interfaces/msg/
 # Copy the IDL
-cp -r /opt/ros/humble/share/builtin_interfaces/msg/Time.idl libraries/AP_DDS_Client/Idl/builtin_interfaces/msg/
-# Now, apply the mods manually to be compliant with MicroXRCEDDSGen limitations
-# Create an output directory to test it
-mkdir -p /tmp/xrce_out
-# Run the generator
-microxrceddsgen -replace -d /tmp/xrce_out libraries/AP_DDS_Client/Idl/builtin_interfaces/msg/Time.idl
-# cat /tmp/xrce_out/
+cp -r /opt/ros/humble/share/builtin_interfaces/msg/Time.idl libraries/AP_DDS/Idl/builtin_interfaces/msg/
```

Comment:
Can we remove the `-r` flag since it's just a single file?

Reply:
yeah will change it now !

Reply:
done